manage.py

```diff
@@ -24,11 +24,30 @@ def db_seed():
     db.create_all()
 
     # seed anything here we might need
-    user = User(username='iandouglas', email='ian.douglas@iandouglas.com')
-    db.session.add(user)
+    item1 = Item(name='beard trimmer', category='toiletries', weather='all', gender='male')
```

Comment:
Seeds! This will make testing a lot easier until we can import the full set of data